drivers/ma_yya_report/app/models/ma_yya_report/report.rb

```diff
@@ -197,7 +197,8 @@ def describe_filter_as_html
 
         D4a: a_t[:pregnant].eq(1).and(a_t[:due_date].gt(report_start_date)).
           or(Arel.sql(custodial_parent_query)).and(a_t[:head_of_household].eq(true)),
-        D4b: a_t[:sexual_orientation].in([2, 3, 4, 5]).or(a_t[:gender].in([5, 6])).and(a_t[:head_of_household].eq(true)),
+        # Why did D4b define LGBTQ differently from G3a? It was including 8/9 gender responses as LGBTQ
+        D4b: lgbtq_query.and(a_t[:head_of_household].eq(true)),
```

Comment:
@awisema or @eanders , any idea whether it was purposeful that D4b (_Number of YYA served who were LGBTQ+_) and G3a (_Number of YYA served who were LGBTQ+_ for youth who transitioned into stabilized housing) had different definitions of LGBTQ? 

Looks like a bug to me. D4b previously included gender "6" which which usually means Questioning, but we overload it in the universe_calculator to include "Doesn't know" and "prefers not to answer" as well.